syncer/syncer.go

```diff
@@ -268,12 +269,12 @@ func (s *Syncer) synchronize(ctx context.Context) bool {
 		close(vQueue)
 		<-vDone
 		s.setStateAfterSync(ctx, success)
-		logger.With().Info(fmt.Sprintf("finished sync run #%v", s.run),
+		logger.With().Debug(fmt.Sprintf("finished sync run #%v", s.run),
```

Comment:
i'd rather this to be the Info and line 304 to be Debug.

Reply:
but no fuss. i can change it after you merge

Reply:
done!